docs/PIPELINE_DEPLOYMENTS.md

```diff
@@ -0,0 +1,169 @@
+# Pipeline deployments
+
+The *Secured Data Warehouse Module* creates a secured Bigquery Data Warehouse infrastructure.
+To use this infrastructure to deploy new Dataflow Flex Pipelines use the instructions in the following sections.
+
+We assume you are familiar with [Deploying a Pipeline](https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline).
+
+## *Secured Data Warehouse Module* deployment
+
+It is necessary configure some controls for the deployment of the Secured Data Warehouse to allow user to create Dataflow Flex pipelines.
+
+The Secured Data Warehouse module uses [VPC Service Controls](https://cloud.google.com/vpc-service-controls/docs/service-perimeters).
+
+The identity deploying the Dataflow pipeline must be in the [access level](https://cloud.google.com/access-context-manager/docs/create-basic-access-level#members-example) of the perimeter.
+You can add it using the input `perimeter_additional_members` of the *Secured Data Warehouse Module*.
+
+To use a private template repository outside of the perimeter, the identity deploying the Dataflow pipeline must be in a *egress rule* that
+allows the Dataflow templates to be fetched. In the *Secured Data Warehouse Module* you configure it using the appropriated list below.
+
+- For the **confidential perimeter**, the identity needs to be added in the input `confidential_data_dataflow_deployer_identities` of the *Secured Data Warehouse Module*.
+- For the **data ingestion perimeter**, the identity needs to be added in the input `data_ingestion_dataflow_deployer_identities` of the *Secured Data Warehouse Module*.
+
+You also need to add the private template repository project to the *egress rule*.
+You must use the variable `sdx_project_number` to provide the *project number* of project.
```

Comment:
that is outside of the perimeter and is hosting your flex template repository.

Can we provide an example gcloud command to get the project number?